cypress/features/regression/test/actionPopover.feature

```diff
@@ -87,51 +87,51 @@ Feature: Action Popover component in noIFrame
 
   @positive
   Scenario: Open Action Popover and close it using Tab key
```

Comment:
Could this scenario and others for keyboard tests be put into an Examples table?

Reply:
as above